addons/mrp/models/stock_rule.py

```diff
@@ -88,3 +88,12 @@ def _push_prepare_move_copy_values(self, move_to_copy, new_date):
         new_move_vals = super(StockRule, self)._push_prepare_move_copy_values(move_to_copy, new_date)
         new_move_vals['production_id'] = False
         return new_move_vals
+
+class ProcurementGroup(models.Model):
+    _inherit = 'procurement.group'
+
+    @api.model
+    def _get_moves_to_assign_domain(self):
+        domain = super(ProcurementGroup, self)._get_moves_to_assign_domain()
+        domain.append(('production_id', '=', False))
+        return domain
```

Comment:
from odoo.osv import expression
domain = super(ProcurementGroup, self)._get_moves_to_assign_domain()
domain = expression.AND([domain, [('production_id', '=', False)]])